src/psm/src/ir_solver.cpp

```diff
@@ -805,15 +755,15 @@ bool IRSolver::CreateGmat(bool connection_only) {
             }
             int x_loc1, x_loc2, y_loc1, y_loc2;
             if (layer_dir == dbTechLayerDir::Value::HORIZONTAL) {
-              y_loc1 = y - y_cut_size / 2;
-              y_loc2 = y + y_cut_size / 2;
-              x_loc1 = x - (x_bottom_enclosure + x_cut_size / 2);
-              x_loc2 = x + (x_bottom_enclosure + x_cut_size / 2);
+              y_loc1 = y - ((num_via_rows-1)*y_cut_spacing + y_cut_size) / 2;
```

Comment:
Why isn't y_cut_size multipied by num_via_rows?  Likewise below

Reply:
Thanks, I mistakenly took the spacing as center to center. This should be fixed in the coming commit.